src/utils/midi.ts

```diff
@@ -41,6 +44,8 @@ export async function playNote(note: MidiNote) {
     if (!(await loadInstrument(note.name))) {
       return await wake()
     }
+  } else {
+    await loadedInstruments.get(note.name)
```

Comment:
Not totally sure, but does anything bad happen here if this promise resolves to false? Should this abort?